README.md

```diff
@@ -8,7 +8,7 @@
 
 # QuickStart
 
-[Download this repo](https://github.com/phase2/pattern-lab-starter/archive/master.zip), rename folder to `patternlab` and place in themes directory if integrating with Drupal 8, and then `cd` into folder before running below commands. To name it something else, see [docs/setup.md](docs/setup.md).
+[Download this repo](https://github.com/phase2/pattern-lab-starter/archive/master.zip), rename folder to `patternlab` and place in themes directory if integrating with Drupal 8, and then `cd` into folder before running below commands.
```

Comment:
Interesting, I hadn't thought about this yet. This solves my q from yesterday about adding theme hooks. There's no naming morass because we're not renaming. Very cool.

Reply:
@jryanconklin, we're definitely going to find an intensely unique token name to make find/replace easier for folks. (thundersnow! JK. Kind of.)